.github/workflows/sykmeldinger-backend-proxy.yaml

```diff
@@ -0,0 +1,84 @@
+name: Bygg sykmeldinger-backend-proxy
+on:
+  push:
+    paths:
+      - sykmeldinger-backend-proxy/**
+      - .github/workflows/sykmeldinger-backend-proxy.yaml
+env:
```

Comment:
Jeg synes kanskje appen bør hete det samme som repoet, særlig siden vi allerede har en app som heter sykmeldinger-backendproxy (tror iallefall det er det den heter 😅 ). 

Skal denne brukes for alt som gjelder sykmeldinger, eller kun for det som eksponeres fra sykmeldinger-backend? :)

Reply:
Sånn vi har det i vårt repo har vi flere proxyer, en backend per app. Men alle ligger i flex-proxy repoet.

Reply:
ja, navngivningen var intensjonell slik håvard beskriver

Reply:
Ah, skjønner :)